script/create-sitemaps.js

```diff
@@ -0,0 +1,37 @@
+const metalsmithSitemap = require('metalsmith-sitemap');
+const sitemap = require('sitemap');
+
+function createSitemaps(BUILD_OPTIONS) {
+  const hostname =
+    BUILD_OPTIONS.host === 'localhost'
+      ? 'http://localhost'
+      : BUILD_OPTIONS.host;
+
+  const metalsmithSitemapOptions = {
+    hostname,
+    omitIndex: true,
```

Comment:
This omitIndex is causing us to skip the homepage for aXe testing, I think. (not saying we should change it here, just pointing it out)